ui.apps/src/main/content/jcr_root/apps/acs-commons/config/org.apache.sling.jcr.repoinit.RepositoryInitializer-aem-cs.config

```diff
@@ -0,0 +1,34 @@
+scripts=[
+"
+create path /content/dam(sling:Folder)
```

Comment:
why do you change ACL creation for paths which can be converted correctly and also don't have any restrictions with the importer user?

Reply:
i dont think i need /content/dam (ill remove, i added since we ACL it, but im not sure how/when its created) - but i was seeing issues w/ content/cq:tags where the ACLs could not be applied to the paths as they didnt exist. /etc/tags made sense to me (since this is an old path) but AFIAK /content/cq:tags should exist - that said, IDK how the product bootstraps that content structure, so maybe my repoinit was triggering before it was created? Otherwise, i was getting ERRORs.